pkg/rancher-desktop/pages/Images.vue

```diff
@@ -133,14 +133,16 @@ export default {
     });
     ipcRenderer.send('settings-read');
 
-    ipcRenderer.on('extensions/list', this.installedExtensionsUpdate);
-    ipcRenderer.send('extensions/list');
+    ipcRenderer.on('extensions/changed', () => {
+      this.$store.dispatch('extensions/fetch');
+    });
+    this.$store.dispatch('extensions/fetch');
   },
   beforeDestroy() {
     ipcRenderer.invoke('images-mounted', false);
     ipcRenderer.removeAllListeners('images-mounted');
     ipcRenderer.removeAllListeners('images-changed');
-    ipcRenderer.removeListener('extensions/list', this.installedExtensionsUpdate);
+    ipcRenderer.removeListener('extensions/changed');
```

Comment:
This seems to be missing the listener to remove.
```
TypeError: The "listener" argument must be of type function. Received undefined
    at __node_internal_captureLargerStackTrace (node:internal/errors:465:5)
    at new NodeError (node:internal/errors:372:5)
    at __node_internal_ (node:internal/validators:238:11)
    at checkListener (node:events:259:3)
    at EventEmitter.removeListener (node:events:690:7)
    at VueComponent.beforeDestroy (Images.vue?b4a2:145:1)
```

Reply:
Right.. I don't know why I modified it this way. Fixed and squashed.